client/my-sites/sharing/buttons/options.jsx

```diff
@@ -211,10 +215,15 @@ const connectComponent = connect(
 		const siteId = getSelectedSiteId( state );
 		const isJetpack = isJetpackSite( state, siteId );
 		const isTwitterButtonAllowed = ! isJetpack || isJetpackMinimumVersion( state, siteId, '3.4-dev' );
+		const postTypes = values( getPostTypes( state, siteId ) );
 
 		return {
+			buttons: getSharingButtons( state, siteId ),
```

Comment:
That sounds fine to me. And in the case of the `<QueryPostTypes />`, since it's only used in `options.jsx`, makes sense that it be the one to render it. 👍